### DISCUSSION TITLE
ChatComposite is missing messages sent when iOS in is background mode

### DISCUSSION THREAD

**Discussion by DercilioFontes** [Azure/communication-ui-library#1705] [General]
Hi,

I would like to know if you guys have a solution to that problem. Everything is working fine, and thanks for the package. 

The only problem I am facing and I couldn't find any help googling is that if the ios side that is chatting goes to background mode it misses the messages sent to it during that time. I know the iOS background mode doesn't allow the communication to happen, but when it is active again it could update with new messages.

I was trying to use some ChatAdapter methods like `fetchInitialData` and `loadPreviousChatMessages` but no result.

I'm in an React Ionic project, using:
    "@azure/communication-chat": "^1.1.2",
    "@azure/communication-common": "^2.0.0",
    "@azure/communication-react": "^1.1.0",

**Comment by ddematheu2** [comment]
hey @DercilioFontes moved this conversation to an issue.

**Reply by ddematheu2** [reply]
https://github.com/Azure/communication-ui-library/issues/1720

**Comment by alkwa-msft** [comment]
Closing this discussion due to inactivity. Feel free to re-open if this is still an issue.
